src/Agent.Worker/Release/ContainerProvider/FileContainerProvider.cs

```diff
@@ -82,15 +82,21 @@ async delegate
 
         public async Task<Stream> GetFileTask(ContainerItem ticketedItem, CancellationToken cancellationToken)
         {
+            this._executionContext.Debug(StringUtil.Loc("RMDownloadGetFileContainerClient", ticketedItem.Path));
```

Comment:
we normally don't localize debug message, since they are not always get print out, and loc process cause money. 💰